engine/unity5/Assets/Scripts/States/MainState.cs

```diff
@@ -182,32 +182,14 @@ public override void Start()
 
                 try
                 {
-                    result = LoadRobot(PlayerPrefs.GetString("simSelectedRobot"), RobotTypeManager.IsMixAndMatch);
+                    result = LoadRobot(PlayerPrefs.GetString("simSelectedRobot"), false);
                 } catch (Exception e) {
                     MonoBehaviour.Destroy(GameObject.Find("Robot"));
                 }
 
                 if (!result)
                 {
-                    /*PlayerPrefs.SetString("simSelectedRobot", robotDirectory + Path.DirectorySeparatorChar + "Dozer" + Path.DirectorySeparatorChar);
-                    if (!LoadRobot(PlayerPrefs.GetString("simSelectedRobot"), RobotTypeManager.IsMixAndMatch))
-                    {
-                        bool loadedBot = false;
-
-                        string[] robots = Directory.GetDirectories(robotDirectory);
-                        foreach (string robot in robots) {
-                            if (LoadRobot(robot, RobotTypeManager.IsMixAndMatch)) {
-                                PlayerPrefs.SetString("simSelectedRobot", robot);
-                                loadedBot = true;
-                            }
-                        }
-
-                        if (!loadedBot) {
-                            AppModel.ErrorToMenu("Could not load any robots in default directory:\n" + robotDirectory);
-                            return;
-                        }
-                    }*/
-                    AppModel.ErrorToMenu("ROBOT_SELECT");
+                    AppModel.ErrorToMenu("ROBOT_SELECT|Could not find the selected bot.");
```

Comment:
To be consistent with `SimUI.cs 490`, change _bot_ to _robot_ and remove the period.